boards/common/arduino-atmega/board.c

```diff
@@ -49,6 +56,12 @@ void board_init(void)
     /* initialize the board LED */
     gpio_init(LED0_PIN, GPIO_OUT);
     LED0_OFF;
+#ifdef CPU_ATMEGA32U4
```

Comment:
IIUC, LED1 and LED2 are specific to the leonardo board but not to ATMEGA32U4 cpu. There are other arduino boards with this CPU but no on-board LED1/LED2 I think.
What about a dedicated board.c for leonardo ?